real_time/surface_src/java_src/BuggyRos/src/main/java/com/roboclub/robobuggy/ros/NodeChannel.java

```diff
@@ -35,40 +37,46 @@
 	ENCODER_RESET("encoder_reset", true),
 	AUTON_BRAKE_STATE("auton_brake_state", true),
 	TELEOP_BRAKE_STATE("teleop_brake_state", true),
+
+	IMU("imu", true),									//The most recent imu measurement
+	HILL_CREST_IMU("hill_crest_imu",true),
+	IMU_MAGNETIC("mag", true),
 	IMU_LINEAR_ACC("imu_linear_acc", true),
-	IMU_LINEAR_NO_GRAV("imu_linear_no_grav", true),
 	IMU_ANG_VEL("imu_ang_vel", true),
-	IMU_TEMP("imu_temp", true),
+	IMU_COMPASS("imu_compass", true),
 	IMU_ANG_POS("imu_ang_pos", true),
+	IMU_ACCELERATION("imu_acceleration", true),
+	IMU_INCLINATION("imu_inclination", true),
+
 	UNKNOWN_CHANNEL("unknown", false),
```

Comment:
yes that is ok for java enums, basically it'll look for another object, see the semicolon, and end. It's a weird little thing I've never really questioned before